Src/FluentAssertions/Primitives/DateTimeOffsetRangeAssertions.cs

```diff
@@ -132,5 +132,5 @@ private static string PositionRelativeToTarget(DateTimeOffset actual, DateTimeOf
 
     /// <inheritdoc/>
     public override bool Equals(object obj) =>
-        throw new NotSupportedException("Calling Equals on Assertion classes is not supported.");
+        throw new NotSupportedException("Equals is not part of Fluent Assertions. Did you mean Before() or After() instead?");
```

Comment:
This one is really unrealistic to happen in real life, but I figure this is an appropriate message for it.

Reply:
With the number of users of Fluent Assertions history has shown that Murphy's Law to calling the wrong method.